docs/tools/behat.md

```diff
@@ -0,0 +1,20 @@
+Behat
+=====
+[Behat](http://behat.org/en/latest/) is installed with the [Behat Drupal Extension](https://behat-drupal-extension.readthedocs.io/en/3.1/) at `vendor/bin/behat`.
+
+Configuration
+-------------
+Behat configuration lives in `behat.yml` in the repository root.  The FeatureContext and all feature definitions live in the `features` directory.
```

Comment:
The tests themseleves - FeatureContext and all feature definitions - live in the `features` directory.